app/views/trainees/show.html.erb

```diff
@@ -47,7 +47,7 @@
     <% end %>
   </ul>
   <div class="govuk-tabs__panel" id="about">
-    <%= render TraineeAbout::View.new(trainee: @trainee, current_user: current_user) %>
+    <%= render TraineeAbout::View.new(trainee: @trainee, current_user: current_user, has_missing_fields: missing_fields&.excluding(Submissions::MissingDataValidator::OPTIONAL_FIELDS).present?) %>
```

Comment:
This could maybe live in a helper?

Reply:
Not sure which part you mean?  

Maybe `missing_fields&.excluding(Submissions::MissingDataValidator::OPTIONAL_FIELDS).present?` ?